tests/unit/games-dao.test.js

```diff
@@ -0,0 +1,215 @@
+const appRoot = require('app-root-path');
+const chai = require('chai');
+const chaiAsPromised = require('chai-as-promised');
+const chaiExclude = require('chai-exclude');
+const _ = require('lodash');
+const proxyquire = require('proxyquire');
+const sinon = require('sinon');
+const { createConnStub } = require('./test-util');
+const testData = require('./test-data');
+
+
+const gamesSerializer = appRoot.require('api/v1/serializers/games-serializer');
+
+chai.should();
+chai.use(chaiExclude);
+chai.use(chaiAsPromised);
+
+let gamesDao;
+
+describe('Test games-dao', () => {
+  const {
+    fakeId,
+    testCases,
+    fakeGamePostBody,
+    fakeGamePatchBody,
+  } = testData;
+
+  beforeEach(() => {
+    const serializeGamesStub = sinon.stub(gamesSerializer, 'serializeGames');
+    const serializeGameStub = sinon.stub(gamesSerializer, 'serializeGame');
+    serializeGamesStub.returnsArg(0);
+    serializeGameStub.returnsArg(0);
+
+    gamesDao = proxyquire(`${appRoot}/api/v1/db/oracledb/game-dao`,
+      {
+        [`${appRoot}/api/v1/serializers/games-serializer`]: {
+          serializeGame: serializeGameStub,
+          serializeGames: serializeGamesStub,
+        },
+      });
+  });
+
+  afterEach(() => sinon.restore());
+
+  describe('Test getGames', () => {
+    const validTestCases = [
+      testCases.emptyResult,
+      testCases.multiResult,
+      testCases.singleResult,
```

Comment:
You should destructure `testCases.emptyResult`, `testCases.multiResult`, `testCases.singleResult` and all other used test cases from `testCases` instead of indexing them every time when you need them.